### PR TITLE
Log the adapter & the routing table by default when config changes.

### PR DESCRIPTION
**Pull request by ozevren**

### CODE REVIEW

**Comment by mandarjog** [comment]
@ozevren this touches files that we are changing for grpc adapters. Let's hold this until that gets in.

**Comment by ozevren** [comment]
Sure. It is not urgent.

**Comment by ozevren** [comment]
@mandarjog I am planning to remove hold on this. Let me know if you're still working on conflicting files.

**Comment by geeknoid** [comment]
Mandar's out for another couple weeks, so I'd go ahead with this checkin.

**Comment by ozevren** [comment]
Waiting for a /lgtm

File: mixer/pkg/runtime/runtime.go [hunk 1]
```diff
@@ -156,17 +156,17 @@ func (c *Runtime) processNewConfig() {
...
-		newHandlers, newSnapshot, builder, c.defaultConfigNamespace, log.DebugEnabled())
+		newHandlers, newSnapshot, builder, c.defaultConfigNamespace, true)
 
 	oldContext := c.dispatcher.ChangeRoute(newRoutes)
 
 	c.handlers = newHandlers
 	c.snapshot = newSnapshot
 
-	log.Debugf("New routes in effect:\n%s", newRoutes)
+	log.Infof("New routes in effect:\n%s\n", newRoutes.String())
```
Comment (geeknoid) [inline mixer/pkg/runtime/runtime.go]: Why does this and the other thing above need to be on by default? This is pollution in the user's log IMO.

Comment (geeknoid) [reply mixer/pkg/runtime/runtime.go]: Better idea would be to put this behind a scope, and then selectively enable that scope.

**Pull request closed without merging**